csm-1.5/CASMTRIAGE-6796-noos-repo/README.md

```diff
@@ -0,0 +1,21 @@
+# CSM 1.5.0 Tarball Noos Repository
+
+This hotfix will:
+* Copy a new `metal-ipxe` package into an extracted CSM tarball.
+* Recreate/update the repodata in the extracted tarball.
+
+After running this hotfix, the tarball can be uploaded to Nexus.
```

Comment:
This should probably say something like `continue installation as usual` instead of `tarball can be uploaded to Nexus`